### PR TITLE
Added config modal to support number of messages and dev/prod mode feature 

### PR DESCRIPTION
**Pull request by aditygrg2**
Related to Issue #24 

Used a Modal component using a slot composition by Svelte. Also formatting is done, and made the request without SSL.

### CODE REVIEW

**Comment by glaslos** [comment]
Please merge the main branch into your feature branch

**Comment by aditygrg2** [comment]
Hi @glaslos, merged. Please review and let me know if any changes are required to improve this contribution

File: src/App.svelte [hunk 1]
```diff
@@ -1,182 +1,268 @@
...
+  import { onMount } from "svelte";
+  import { now } from "svelte/internal";
 
-    import Content from './Content.svelte';
-    import Message from './Message.svelte';
-    import SSOButton from './SOOButton.svelte';
-    import LogoutButton from './LogoutButton.svelte';
-    import SOORevokeButton from './SOORevokeButton.svelte';
-    import { validate } from './session';
+  import Modal from "./Modal.svelte";
```
Comment (glaslos) [inline src/App.svelte:5]: Are you using prettier?

Comment (aditygrg2) [reply src/App.svelte]: I used a different formatter maybe. Now, the code is updated according to the main prettierrc file. @glaslos

**Comment by glaslos** [comment]
We should exclude the package-lock.json from prettier

**Comment by aditygrg2** [comment]
> We should exclude the package-lock.json from prettier

Updated .prettierignore file to include package-lock.json, and package.json. Also, I was not using any json formatter for formatting these two files.

File: package-lock.json
Comment (glaslos) [inline package-lock.json:1]: Still modified this page. Do a checkout of the previous version 🤔

**Comment by aditygrg2** [comment]
@glaslos updated the files. Please check and thank you for several re-reviews.

**Comment by glaslos** [comment]
@aditygrg2 please resolve the conflicts.

**Comment by aditygrg2** [comment]
Hi @glaslos, conflicts are resolved now.

**Review by glaslos — APPROVED** [review]
Nice work, especially for keeping it simple and on target!

**Pull request merged**